github-issues.go

```diff
@@ -183,6 +186,7 @@ func populateIssues(url string) {
 	json.Unmarshal(htmlData, &mIssues)
 
 	var flag int
+	const layout = "Jan 2, 2006 at 3:04pm (PST)"
```

Comment:
This is incorrect you should use either one of the following time formats which are standard and equivalent.

 - http.TimeFormat
 - time.RFC1123